src/transformers/diffLineTransformer.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+const { highlightDiffLine } = require('./transformerUtils');
+
+function createDiffLineTransformer() {
+  // How do I get the parsed code fence options here?
+  /** @type {LineTransformer<any>} */
+  const diffLineTransformer = ({ line, meta: { diff } }) => {
+    if (diff && line.text.startsWith('+ ')) {
+      line.text = line.text.replace(/^\+ /, '');
```

Comment:
I _think_ it doesn’t matter at this point, but it would probably be best to avoid mutating the input parameters. I should type them as `readonly`, in fact.

Reply:
Changing to:

```js
const text = line.text.replace(/^\+ /, '');
return {
  line: highlightDiffLine({ ...line, text }, 'add'),
```